src/documents/IoCReferences.md

```diff
@@ -220,7 +220,7 @@ fluid.defaults("fluid.moduleLayoutHandler", {
 });
 ```
 
-## Examples of `{<iocss expression>}` ##
+## Examples of {<iocss expression>} ##
```

Comment:
This doesn't render properly - the angle brackets and their content are stripped.